### ISSUE TITLE
Replacement param

### ISSUE THREAD

**Issue by johnnydimas** [tunnckoCore/randomorg-js#9]
Hi there,

It does not seem the 'replacement' parameter with generate integer is being recognized. 

I am using the standard example:

```
var result = randomJs
.apikey('xxxxxxxxxx')
.headers({'User-Agent': 'application/json'})
.method('generateIntegers')
.params({n:10, min: 0, max: 10, replacement: false})
.post(function(error, stream, body) {
      var results = body.result.random.data;
      console.log(results);
});
```

Am I doing something wrong? Thanks!

**Comment by tunnckoCore** [comment]
> Am I doing something wrong?

I don't think so. Library have big need to be refactored. Current problem maybe comes from that it's not used merging/extending for objects and defaults.

PRs welcome, i don't have atm to start the refactor.

**Comment by bcorob** [comment]
It seems all optional parameters are ignored.  I noticed the 'format' parameter doesn't work when specified either.

It looks like yuezk has resolved the issue and submitted PR #10 to address this bug.  His change works for me, can you please review and accept?

**Comment by tunnckoCore** [comment]
@bcorob hey thanks! i just didn't notice it. That repo is quite old, it definitely deserve some refactoring.
